torch/linalg/__init__.py

```diff
@@ -154,26 +164,28 @@
 """)
 
 det = _add_docstr(_linalg.linalg_det, r"""
-linalg.det(input, *, out=None) -> Tensor
+linalg.det(A, *, out=None) -> Tensor
 
-Computes the determinant of a square matrix. Supports batched inputs.
+Computes the determinant of a square matrix.
 
-Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
 
 .. note:: This function is computed using :func:`torch.lu`.
           For CUDA inputs, this function synchronizes that device with the CPU.
 
-.. note:: Backward through `det` internally uses :func:`torch.linalg.svd` when :attr:`input` is not
-          invertible. In this case, double backward through `det` will be unstable when :attr:`input`
+.. note:: Backward through `det` internally uses :func:`torch.linalg.svd` when :attr:`A` is not
```

Comment:
Let's avoid use of the word "backward" (we're supporting forward AD soon!) and use phrases like "The gradient of..." So maybe in this case:

"The gradient of \`det\` is computed using ..."
"Because of this the gradient of the gradient of \`det\` is unstable when :attr:\`A\` doesn't have unique singular values..."

By "unstable" here, do we mean that the computation is incredibly numerically sensitive, or do we mean it's inaccurate?

Reply:
The "backward" point is a good one. I'll implement it throughout the docs.

About this note, I'm going to delete it, as it will become obsolete once Nik implements the new formula for the backward, which should be in soon.